core/src/main/resources/error/error-classes.json

```diff
@@ -167,5 +167,89 @@
   "WRITING_JOB_ABORTED" : {
     "message" : [ "Writing job aborted" ],
     "sqlState" : "40000"
+  },
+  "UNABLE_TO_DELETE_PARTITION_PATH" : {
```

Comment:
I see one case of '%s' and most without single quotes.
Maybe standardizing on single quoting all strings would be useful.